servers/republik/modules/crowdfundings/graphql/schema-types.js

```diff
@@ -413,4 +416,11 @@ type PostfinancePayments {
   items: [PostfinancePayment!]!
   count: Int!
 }
+
+type MembershipPotStats {
```

Comment:
Would love to see some documentation on this as I fail to grasp it, in particular: difference between donated and generated memberships, and also what is surplus for?